js/components/main.js

```diff
@@ -102,6 +102,19 @@ class Main extends ImmutableComponent {
     // TODO
   }
 
+  onHamburgerMenu () {
+    let settings = {}
+    const resourceNames = ['httpsEverywhere', 'adblock', 'trackingProtection', 'adInsertion']
```

Comment:
would be better if we pulled these in from a constants file in case the names ever change.